src/test/java/com/store/StockStoreTest.java

```diff
@@ -0,0 +1,168 @@
+package com.store;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.log4j.Logger;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static com.store.StockStore.PATCH_TO_FILES;
+
+public class StockStoreTest {
+    private static final Logger LOGGER = Logger.getLogger(StockStoreTest.class);
+    private ObjectMapper mapper;
+
+    @Before
+    public void setUp() {
+        mapper = new ObjectMapper();
+    }
+
+    @Test
+    public void addFruits() {
+        clear();
+        StockStore flowersStore = new StockStore();
```

Comment:
better add this line and next to method setUp()
and maybe adding of fruits

Reply:
add `        StockStore flowersStore = new StockStore();
        List<Fruit> fruits = new ArrayList<>();` to setUp()